src/shared/sam/activation.ts

```diff
@@ -132,6 +135,24 @@ async function activateCodeLensProviders(
         telemetryService
     }
 
+    disposables.push(
+        vscode.languages.registerCodeLensProvider(
+            [
+                {
+                    language: 'yaml',
+                    scheme: 'file',
+                    pattern: '**/*template.{yml,yaml}'
+                },
+                {
+                    language: 'json',
+                    scheme: 'file',
+                    pattern: '**/*template.json'
```

Comment:
Does the existing local debug functionality support json files? If so, thats great (and a new feature for the toolkit), if not, we may have to restrict this to yaml only, until json is supported.

Reply:
Good point, it does not. I added issue #976 to track impl of json. I think it's something we should support. Seems low cost.